src/python/bot/tasks/corpus_pruning_task.py

```diff
@@ -682,23 +684,26 @@ def do_corpus_pruning(context, last_execution_failed, revision):
 
   logs.log('Finished.')
 
-  result = CorpusPruningResult(
-      coverage_info=coverage_info,
-      crashes=list(crashes.values()),
-      fuzzer_binary_name=fuzzer_binary_name,
-      revision=environment.get_value('APP_REVISION'))
-
   sources = ','.join([
       fuzzer.fuzz_target.project_qualified_name()
       for fuzzer in context.cross_pollinate_fuzzers
   ])
 
-  record_cross_pollination_stats(
-      pruner_stats, pollinator_stats, project_qualified_name, sources,
-      context.tag, pre_pollination_corpus_size, minimized_corpus_size_units,
-      context.cross_pollination_method)
-
-  return result
+  if not pruner_stats or not pollinator_stats:
```

Comment:
nit: Start your if with the positive case, to avoid a mental double negative when reasoning about the "else".

This can be further simplified to e.g. 

```python

cross_pollination_stats = None
if pruner_stats and pollinator_stats:
  cross_pollination_stats = ...

```